pom.xml

```diff
@@ -379,12 +379,12 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.88</minimum>
+                                            <minimum>0.71</minimum>
                                         </limit>
                                         <limit>
                                             <counter>BRANCH</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.83</minimum>
+                                            <minimum>0.59</minimum>
```

Comment:
Why was this lowered so much?

Reply:
It was never checking and enforcing it before. This coverage number is specific to the integ test; it doesn't include what is covered by unit tests.

Reply:
I think the integration tests where in the wrong namespace which was why the coverage was high since we have unit tests also in the same namespace.

Reply:
_This_ coverage was never evaluated. The unit test coverage was including coverage from the integ tests.